shell/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -672,7 +672,7 @@ AVMediaType ParseMediaType(const std::string& media_type) {
     if (args->GetNext(&appearance)) {
       [[NSApplication sharedApplication] setAppearance:appearance];
     } else {
-      args->ThrowError("Invalid app appearance provided as first argument");
+      args->ThrowTypeError("Invalid app appearance type provided.");
```

Comment:
I really don't like these kinds of errors, it makes it hard to debug what's actually gone wrong.

E.g. if the error was `Invalid app appearance type "BadThing" provided, expected "light" or "dark"`